### PR TITLE
Add image classification and autograd to pip integration

### PR DESCRIPTION
**Pull request by Deliadd**
add imageclassification

### CODE REVIEW

File: pyzoo/zoo/examples/run-example-tests-pip.sh [hunk 1]
```diff
@@ -46,5 +46,25 @@ now=$(date "+%s")
...
+then
+    clear_up
+    echo "imageclassification failed"
+    exit $exit_status
+fi
+
+unset SPARK_DRIVER_MEMORY
+now=$(date "+%s")
+time2=$((now-start))
+
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-tests-pip.sh]: add `echo "imageclassification time used:$time2 seconds"`?

**Comment by Deliadd** [comment]
successful in Jenkins @hkvision

**Comment by Deliadd** [comment]
ZOO-PR-Pip-Example-Integration#24

File: pyzoo/zoo/examples/run-example-tests-pip.sh [hunk 2]
```diff
@@ -46,5 +46,58 @@ now=$(date "+%s")
...
+time2=$((now-start))
+echo "imageclassification time used:$time2 seconds"
+
+echo "start example test for autograd"
+#timer
+start=$(date "+%s")
+
+export SPARK_DRIVER_MEMORY=2g
+python ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/autograd/custom.py 
+python ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/autograd/customloss.py
```
Comment (hkvision) [inline pyzoo/zoo/examples/run-example-tests-pip.sh]: separate the test for these two files

**Comment by Deliadd** [comment]
Jenkins pass，ZOO-PR-Pip-Example-Integration#30

**Pull request merged**